README.md

```diff
@@ -7,9 +7,48 @@ Covata Delta provides an easy to use framework for sharing secrets across networ
 
 ## Requirements
 
-- Python 2.7 +
-- Python 3.3 +
+- Python 2.7 + or 3.3 +
+- pip 9.0.1 +
+- virtualenv: `sudo pip install virtualenv`
+- pybuilder: `pip install pybuilder`
 
```

Comment:
Ah good catch! Sorry this should be "Setting up Virtual Environment", and it should actually belong as the first section of in quick start. I believe I don't need to split this to bullet points?